File: Java/GsonTest/src/main/java/Person.java
public class Person {
    String name;
    int age;
    String maritalStatus;

    public Person(String name, int age, String maritalStatus) {
        this.name = name;
        this.age = age;
        this.maritalStatus = maritalStatus;
    }

    public String getName() {
        return name;
    }

    public int getAge() {
        return age;
    }

    public String getMaritalStatus() {
        return maritalStatus;
    }
}

File: PHP/tests/Person.php
<?php

class Person
{
    private $name;
    private $age;

    function __construct($name, $age) {
        $this->name = $name;
        $this->age = (int)$age;
    }

    /** @return string */
    public function getName() {
        return $this->name;
    }

    /** @return int */
    public function getAge() {
        return $this->age;
    }


}